include/gz/sensors/Sensor.hh

```diff
@@ -231,6 +232,30 @@ namespace gz
       /// \return True if there are subscribers, false otherwise
       public: virtual bool HasConnections() const;
 
+      /// \brief Set the sensor in triggered mode. In this mode,
+      /// - the sensor will only update if a new message has been published to
+      ///    the passed _trigger_topic since the last update,
+      /// - until the next message is published on _trigger_topic, all Update
+      ///   calls will return false, and
+      /// - update_rate is forced to zero.
+      /// \param[in] _trigger_topic The topic on which the sensor will listen
+      /// for trigger messages.
+      /// \return True if the sensor was successfully set to triggered mode.
+      public: bool EnableTriggered(const std::string &_trigger_topic);
```

Comment:
camelCase for variable names: `_trigger_topic` -> `_triggerTopic`

Reply:
Done.